src/backend/aspen/database/models/usergroup.py

```diff
@@ -103,7 +103,7 @@ class UserRole(base):  # type: ignore
     """User role grants"""
 
     __tablename__ = "user_roles"
-    role_id = Column(Integer, ForeignKey(Role.id), nullable=False)
+    role_id = Column(Integer, ForeignKey(Role.id), nullable=False, primary_key=True)
```

Comment:
Primary key for this table is a `btree (user_id, group_id)`, is there a missing migration to this PR?

Reply:
I checked every environment, and this is what I see:
```  
"user_roles_pkey" PRIMARY KEY, btree (role_id, user_id, group_id)
```

And also the migration file that creates the user_roles table is what established this primary key.

Reply:
i'm confused, how can the primary key be `btree (role_id, user_id, group_id)`, but then you set the primary_key as just `role_id` above? Is this some sqlalchemy wierdness?

Reply:
There are 3 properties/columns in this model/table with the primary key attribute set to true now, where there were only two before:
https://github.com/chanzuckerberg/czgenepi/blob/867ee81e49da4f15695c6a0e4c7ec9b041900d1b/src/backend/aspen/database/models/usergroup.py#L102-L111

Reply:
I see, my local machine is off for some reason 🥴